### ISSUE TITLE
[ChannelTabs] Request: Lock tabs

### ISSUE THREAD

**Issue by flamedance58** [samfundev/BetterDiscordStuff#21]
**Is your feature request related to a problem? Please describe.**
To prevent accidentally closing tabs that we want to always be there.

**Describe the solution you'd like**
A way to "Lock" the tab that would stop it from being closed until it's unlocked.
Maybe if possible a way to stop it from navigating away from that specific DM/channel/server.

**Describe alternatives you've considered**
none

**Additional context**
none

**Comment by Kamiikaze** [comment]
Sounds like a good feature. The lock would be easy but preventing navigation could maybe be hard to implement.

**Comment by flamedance58** [comment]
> Sounds like a good feature. The lock would be easy but preventing navigation could maybe be hard to implement.

Yah the "prevent navigation" is def the "if it's possible that'd be cool addition to the idea" but I know it'd be tricky if not possibly not doable.

**Comment by Kamiikaze** [comment]
> > Sounds like a good feature. The lock would be easy but preventing navigation could maybe be hard to implement.
> 
> Yah the "prevent navigation" is def the "if it's possible that'd be cool addition to the idea" but I know it'd be tricky if not possibly not doable.

yes. Not familiar with BetterDiscord API, but it would need an event listener which would trigger on url change to change the url to the previous one.
That would be easier than preventing links to navigate.
